CMakeLists.txt

```diff
@@ -1,22 +1,19 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.12)
 
 project(LabRecorder
 	DESCRIPTION "Record and write LabStreamingLayer streams to an XDF file"
 	HOMEPAGE_URL "https://github.com/labstreaminglayer/App-LabRecorder/"
 	LANGUAGES C CXX
 	VERSION 1.14.0)
 
-find_package(LSL REQUIRED
-	HINTS ${LSL_INSTALL_ROOT}
-	"${CMAKE_CURRENT_LIST_DIR}/../../LSL/liblsl/build/"
-	"${CMAKE_CURRENT_LIST_DIR}/../../LSL/liblsl/build/install"
-	"${CMAKE_CURRENT_LIST_DIR}/../../LSL/liblsl/out/build/x64-Release"
-	"${CMAKE_CURRENT_LIST_DIR}/../../LSL/liblsl/out/install/x64-Release"
-	PATH_SUFFIXES share/LSL
-)
+set(QT6_FIRST OFF)
+if(ANDROID)
+	set(QT5_FIRST ON)
```

Comment:
Doesn't Android want QT6_FIRST?

Reply:
Yes, I had to switch the order around several times and missed the QT5. I've pushed a change to the liblsl LSLCMake module so deploying works with Qt6, so once that's released Qt6 can be tried first by default.